Frameworks/Foundation/NSFileManager.mm

```diff
@@ -436,36 +436,48 @@ - (BOOL)copyItemAtPath:(id)srcPath toPath:(id)destPath error:(NSError**)error {
 
     TraceVerbose(TAG, L"Done copying");
 
-    return TRUE;
+    return YES;
 }
 
 /**
- @Status Stub
+ @Status Caveat
+ @Notes does not support file manager move delegates for shouldMoveItem or shouldProceedAfterError.
 */
 - (BOOL)moveItemAtURL:(NSURL*)srcURL toURL:(NSURL*)dstURL error:(NSError**)error {
-    UNIMPLEMENTED();
-
-    return NO;
+    return [self moveItemAtPath:[srcURL path] toPath:[dstURL path] error:error];
```

Comment:
This might do well to check that the file URLs are file URLs; it doesn't make sense to move anything else.